package.json

```diff
@@ -3,10 +3,12 @@
   "version": "0.5.1",
   "description": "Extensions that augments several Mozilla websites (Bugzilla, DXR, Searchfox, MozReview) with code coverage information.",
   "dependencies": {
+    "eslint-plugin-mozilla": "^0.9.0",
     "web-ext": "^2.4.0"
   },
   "devDependencies": {},
   "scripts": {
+    "lint": "./node_modules/.bin/eslint **/*.js",
```

Comment:
You might be able to do just `eslint **/*.js` instead.

Reply:
From what I experienced, I can't call ```eslint``` alone if its not globally installed.

Reply:
Just tested, it works without the path, I tried when it wasn't in the package.json, my bad, I will remove the path.